src/components/Dialog/src/DialogLayer.vue

```diff
@@ -165,6 +165,12 @@ export default {
 	}
 }
 
+@media screen and (--for-tablet-landscape-down) {
```

Comment:
for CSS maintainability reasons we try to write all mobiles styles as the "default" styles and then progressively add the overrides for the tablet & up resolutions. is there a way we can re-write this style so that we have this at the root level:

```css
.DialogContentWrapper {
    width: 100%;
}
```

and then a media query which overrides it to something else for tablet & up like:

```css
@media screen and (--for-tablet-landscape-up) {
    .DialogContentWrapper {
        width: /* something here, like initial, or reset, or idk */;
     ]
}
```

Reply:
it's our bad for having a `--for-tablet-landscape-down` in the first place, I'll look into ways we can get rid of that.